torch/_inductor/codegen/rocm/ck_universal_gemm_template.py

```diff
@@ -251,10 +280,20 @@ def render(self, kernel: ROCmTemplateKernel, op: "CKGemmOperation", **kwargs) ->
         template_buffer_node = kwargs.get("template_buffer_node", None)
         if template_buffer_node is not None:
             self.output_node = template_buffer_node
-        instance_definition, instance_type = self.emit_ck_instance(op)
         X, W = self.input_nodes[0], self.input_nodes[1]
         Y = self.output_node
-        Bias = None  # TBD support gemm_bias
+        Bias = self.input_nodes[2] if 3 == len(self.input_nodes) else None
+
+        op = copy.deepcopy(op)
+
+        if Bias:
+            op.ds_layouts = (torch_layout_to_ck_layout(Bias.get_layout()),)
+            op.ds_element_dtypes = ((self._TORCH_DTYPE_TO_CK[Bias.get_layout().dtype]),)
+            op.c_elementwise_op = "ScaledAdd"
```

Comment:
Genericness of this implies that we can, potentially, support more elaborate epilogue fusion?

Reply:
Yes, the template can support at least elementwise fusion, need to think about more complex patterns